src/main/java/reactor/util/function/Tuple2.java

```diff
@@ -120,8 +120,7 @@ public boolean equals(Object o) {
 
```

Comment:
You might want to replace `equals()` as well:
```
public boolean equals(Object o) {
        if (this == o) return true;
        
        if (o == null || getClass() != o.getClass()) return false;

        Tuple2<?, ?> tuple2 = (Tuple2<?, ?>) o;

        if (size() != tuple2.size()) return false;
        
        if (t1 != null ? !t1.equals(tuple2.t1) : tuple2.t1 != null) return false;
        
        return t2 != null ? t2.equals(tuple2.t2) : tuple2.t2 == null;

    }
```
as well.  (Original version doesn't test == to start with.)